### DISCUSSION TITLE
how to load conftest.pyc file

### DISCUSSION THREAD

**Discussion by tjhlovexyb** [pytest-dev/pytest#9980] [Q&A] [answered]
I'd compiled my project, so conftest.py become conftest.pyc. However, it can not collect my testcases.
<img width="939" alt="image" src="https://user-images.githubusercontent.com/40117858/169475798-37d6dc53-fd7c-4805-b78d-849b9bf19e64.png">

It looks like this in `_pytest/config/__init__.py`:
```
       for parent in reversed((directory, *directory.parents)):
            if self._is_in_confcutdir(parent):
                conftestpath = parent / "conftest.py"
                if conftestpath.is_file():
                    mod = self._importconftest(conftestpath, importmode, rootpath)
                    clist.append(mod)
```
can not load conftest.pyc file.

Is there anyway to solve this problem? Thanks !

**Comment by RonnyPfannschmidt** [comment, marked as answer]
This currently is entirely unsupported

Pytest itself operates in terms of python files, not compiled modules

It's currently not a topic core itself can take as its likely full of "papercuts" and possibly requiring a careful audit
